### DISCUSSION TITLE
Object wysiwyg in Iframe to get frontend styling?

### DISCUSSION THREAD

**Discussion by Daniel-Ateles** [pimcore/pimcore#11730] [Ideas]
Hi,

Why isn't the object WYSIWYG-editor inside an Iframe like the document editor so you get the correct frontend css and thereby see what you get while editing?

I know you can duplicate css from the frontend to editmode to get somewhat similar visual appearance while editing in editmode but I have a proposal of modifying the wysiwygeditor so it loads in an iframe with the complete frontend css, hence circumventing the need to have same/similar css in two places. 

If this have already proven to not be doable due to, for me, unforseen technicalities, or because it is already in the pipline for future releases please let me know, otherwise I would like to have a go at it.

**Comment by johanlindgren** [comment]
Maybe the framework could provide a way to use your own logic to choose css file based on logic in your own function override in pimcore.object.tags.wysiwyg making it similar to the current possibilities to set a custom CKEditor config?
